doc/itof/windows_build_instructions.md

```diff
@@ -51,6 +51,8 @@ cmake --build . --target install --config Release
 ```console
 git clone --branch v5.0.0 https://github.com/analogdevicesinc/ToF
 cd ToF
+git submodule ini
```

Comment:
There is no 'ini' command. It should be 'init'.

Reply:
Typo corrected